uat/test_uat_build.py

```diff
@@ -104,3 +107,75 @@ def test_build_template_maven(change_test_dir):
     check_build_template = sp.run(["gdk", "component", "build"])
     assert check_build_template.returncode == 0
     assert Path(path_HelloWorld).joinpath("greengrass-build").resolve().exists()
+
+
+def test_build_template_gradle_multi_project(change_test_dir):
+    path_multi_gradle_project = Path(change_test_dir).joinpath("gradle-build-test").resolve()
+    zip_file = "gradle-build-test.zip"
+    region = "us-east-1"
+    s3_cl = t_utils.create_s3_client(region)
+    account = t_utils.get_acc_num(region)
+
+    s3_cl.download_file(
+        f"gdk-cli-uat-{region}-{account}",
+        f"do-not-delete-test-data/{zip_file}",
+        str(Path(change_test_dir).joinpath(zip_file).resolve()),
+    )
+    shutil.unpack_archive(
+        Path(change_test_dir).joinpath(zip_file),
+        path_multi_gradle_project,
+        "zip",
+    )
+    os.remove(Path(change_test_dir).joinpath(zip_file))
+    os.chdir(path_multi_gradle_project)
+    assert Path(path_multi_gradle_project).joinpath("recipe.yaml").resolve().exists()
+    config_file = Path(path_multi_gradle_project).joinpath("gdk-config.json").resolve()
+    assert config_file.exists()
+
+    # Update gdk-config file mandatory field like region.
```

Comment:
can we use the util method here to update config file?

Reply:
util method had extra fields which are not needed here. But updating it for consistency.